hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/SerialConsumer.java

```diff
@@ -141,6 +142,9 @@ public Subscription getSubscription() {
     public void updateSubscription(Subscription newSubscription) {
         rateLimiter.updateSubscription(newSubscription);
         sender.updateSubscription(newSubscription);
+        if (messageReceiver != null) {
```

Comment:
Heh this was some dirty and unexpected fix after rebasing on develop. I'll fix this right away.